common/scripts/run_rush_change.py

```diff
@@ -16,9 +16,9 @@
 
 # Verifying with rush change requires the branch that is being merged into to be provided.  More details, https://rushjs.io/pages/commands/rush_change/.
 # With release/* being a potential target branch in addition to master, special case those branches.
-if targetBranch.find("refs/heads/release") != -1:
+if "refs/heads/release" in targetBranch:
     branchCmd = ["-b", targetBranch.replace("refs/heads/", "origin/")]
-elif targetBranch.find("release") != -1:
+elif "release" in targetBranch or targetBranch == srcBranch:
```

Comment:
These changes look fine to me, but I am curious what the situation will be where the target branch equals the source branch.

Reply:
for imodel02 addon validation, where theres a change in the native side, but none on the js side.

essentially dont care for running rush change there